catavatar/catavatar.php

```diff
@@ -18,48 +18,58 @@
 /**
  * Installs the addon hook
  */
-function catavatar_install() {
+function catavatar_install()
+{
 	Addon::registerHook('avatar_lookup', 'addon/catavatar/catavatar.php', 'catavatar_lookup');
 	Addon::registerHook('addon_settings', 'addon/catavatar/catavatar.php', 'catavatar_addon_settings');
 	Addon::registerHook('addon_settings_post', 'addon/catavatar/catavatar.php', 'catavatar_addon_settings_post');
 
-	logger("registered catavatar");
+	logger('registered catavatar');
 }
 
 /**
  * Removes the addon hook
  */
-function catavatar_uninstall() {
+function catavatar_uninstall()
+{
 	Addon::unregisterHook('avatar_lookup', 'addon/catavatar/catavatar.php', 'catavatar_lookup');
 	Addon::unregisterHook('addon_settings', 'addon/catavatar/catavatar.php', 'catavatar_addon_settings');
 	Addon::unregisterHook('addon_settings_post', 'addon/catavatar/catavatar.php', 'catavatar_addon_settings_post');
-	
-	logger("unregistered catavatar");
-}
 
+	logger('unregistered catavatar');
+}
 
-function catavatar_addon_settings(&$a, &$s) {
-	if(! local_user())
+/**
+ * Cat avatar user settings page
+ */
+function catavatar_addon_settings(&$a, &$s)
```

Comment:
Obsolete: Please use the following argument list instead: `(\Friendica\App $a, &$s)`

Reply:
Absolutely, I didn't want to suggest a form that wouldn't work out of the box.